src/App.js

```diff
@@ -13,10 +17,78 @@ function App() {
   // the "5" button, or the operator if they click one of those buttons) and then call your setter function to update state.
   // Don't forget to pass the functions (and any additional data needed) to the components as props
 
+  // State and function for number additions in display.
+  const [numeric, setNumeric] = useState('');
+  const numericString = () => setNumeric(numeric);
+
+  // mathematic operations state (returns an operator string).
+  const [operation, setOperation] = useState();
+
+  // Will need to create an object state that can contain two numeric values and an operations value.
+  const [calculations, setCalculations] = useState([]);
+
+  console.log(calculations);
+
+  // mathematic operations: I'm not really sure if this function will work with the logic 
+  // of the application structor.
+  function mathematics( numericOne, operation, numericTwo) {
+    let total = 0
+    switch(operation) {
+      case '/': 
+        total = numericOne/numericTwo; 
+        break;
+      
+      case "x":
+        total = numericOne/numericTwo;
```

Comment:
I believe you have a type here :), it should be  **numericOne * numericTwo**, 
anyways you're great and I learned from your code stretch code.

Reply:
Thanks so much man! I fixed this problem in a repl.it but forgot to send the change to the actual repository. Oh man though! This project was brutal. Half of the props in my App component could be deleted XD.